sanssouci/post_hoc_bounds.py

```diff
@@ -38,33 +38,34 @@ def max_fp(p_values, thr):
         Annals of Statistics, 48(3), 1281-1303.
     """
 
-    # make sure that thr is sorted
-    if np.linalg.norm(thr - thr[np.argsort(thr)]) > 0.0001:
-        thr = thr[np.argsort(thr)]
-        print("The input thr was not sorted -> this is done now")
+    # make sure that thresholds is sorted
+    if np.linalg.norm(thresholds - np.sort(thresholds)) \
+       > 0.0001:
+        thresholds = np.sort(thresholds)
+        print("The input 'thresholds' was not sorted -> this is done now")
 
     # do the job
-    nS = p_values.shape[0]
-    K = thr.shape[0]
-    size = np.min([nS, K])
+    subset_size = p_values.shape[0]
+    template_size = thresholds.shape[0]
+    size = np.min([subset_size, template_size])
 
     if size < 1:
         return 0
 
-    seqK = np.arange(size)
+    seq_k = np.arange(size)
```

Comment:
The naming could still be improved: `seq_k` is both unclear and hard to pronounce.